docker/entrypoint.sh

```diff
@@ -23,6 +23,10 @@ case "$1" in
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--workspace" "--" "--include-ignored"
         elif [[ "$TEST_FULL_SYNC" -eq "1" ]]; then
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--test" "acceptance" "--" "--nocapture" "--ignored" "full_sync_mainnet"
+        elif [[ "$TEST_DISK_REBUILD" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,,test_sync_to_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "sync_to_mandatory_checkpoint_${NETWORK }"
+        elif [[ "$TEST_CHECKPOINT_SYNC" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,,test_sync_past_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "test_sync_past_mandatory_checkpoint_${NETWORK }"
```

Comment:
Possible Bug: duplicate commas in feature list

```suggestion
            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,test_sync_to_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "sync_to_mandatory_checkpoint_${NETWORK }"
        elif [[ "$TEST_CHECKPOINT_SYNC" -eq "1" ]]; then
            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,test_sync_past_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "test_sync_past_mandatory_checkpoint_${NETWORK }"
```

Reply:
Fixed by: [fix(rw): minor fixes](https://github.com/ZcashFoundation/zebra/pull/4271/commits/044e944715ff59bccd0eb2f415f9a4737cfb336a)